Symfony/CS/Console/Command/FixCommand.php

```diff
@@ -501,7 +501,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
             }
         }
 
-        return empty($changed) ? 0 : 1;
+        return $resolver->isDryRun() || empty($changed) ? 0 : 2;
```

Comment:
@gharlan 

I believe this will return `0` if it was a dry-run but there were failures?

I'm always running this with `--dry-run` on Travis, and I believe it shouldn't return `0` when there were failures, regardless of it is a dry run or not.

What do you think?